app/scripts/modules/kubernetes/src/serverGroup/details/details.html

```diff
@@ -191,7 +191,7 @@ <h4 class="text-center" ng-if="serverGroup.isDisabled">[SERVER GROUP IS DISABLED
     </collapsible-section>
     <collapsible-section heading="Events">
       <div ng-if="serverGroup.events && serverGroup.events.length > 0">
-        <kubernetes-event ng-repeat="event in serverGroup.events" event="event"></kubernetes-event>
+        <kubernetes-event ng-repeat="event in serverGroup.events" event="event"/>
```

Comment:
this needs to be reverted because self closing custom tags are not valid html

```suggestion
        <kubernetes-event ng-repeat="event in serverGroup.events" event="event"></kubernetes-event>
```

Reply:
Ooops, that was my IDE'd doing :\